parser/prism/Translator.cc

```diff
@@ -196,7 +196,12 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             auto backReferenceReadNode = down_cast<pm_back_reference_read_node>(node);
```

Comment:
I don't remember exactly what these were, or how they worked. Could you clarify with a comment on this line?

What's an example of a back reference?